external-builds/pytorch/ptbuild.py

```diff
@@ -295,7 +311,7 @@ def add_common(command_parser: argparse.ArgumentParser):
         )
 
     p = argparse.ArgumentParser("ptbuild.py")
-    default_tag = "v2.6.0"
+    default_tag = "v2.7.0-rc10"
```

Comment:
Let's keep the default at 2.6.0 until we get the aotriton patch upstream. We do have people using that.

Reply:
I've amended the PR to change `default_tag` back to `v2.6.0` in https://github.com/ROCm/TheRock/compare/4e0c40671f8902ab320f8f53f51fb5b37370927c..1e8847e6a40736bacd4ae10b68c8baa748e20184